charts/trustify/templates/helpers/_infrastructure.tpl

```diff
@@ -38,17 +38,19 @@ Arguments (dict):
 {{- if eq ( include "trustification.application.metrics.enabled" .root ) "true" }}
 - name: METRICS
   value: "enabled"
+- name: OTEL_METRIC_EXPORT_INTERVAL
```

Comment:
I think we need a way to handle if this variable is not set.

Reply:
got it, sorry I saw the comment now :+1:

Reply:
cool this is working:

<img width="1918" height="231" alt="2025-08-06_08-12" src="https://github.com/user-attachments/assets/b87c1525-cda3-408d-80a5-36a9ab59b1d1" />
<img width="1920" height="204" alt="2025-08-06_08-13" src="https://github.com/user-attachments/assets/b5b3bda2-4dde-4fab-b06b-e013ad0b721e" />